apps/cowswap-frontend/src/modules/tradeQuote/hooks/useSmartSlippageFromQuote.ts

```diff
@@ -0,0 +1,9 @@
+import { MAX_SLIPPAGE_BPS } from '@cowprotocol/common-const'
+
+import { useTradeQuote } from './useTradeQuote'
+
+export const useSmartSlippageFromQuote = (): number | null => {
```

Comment:
We should not always use suggested slippage from quote.
For example, when user manually set a slippage in settings, we should use ths slippage from settings.